### DISCUSSION TITLE
iCasa light bulbs flashing

### DISCUSSION THREAD

**Discussion by EvertDekker** [Koenkk/zigbee2mqtt#13665] [Devices]
Hi,
I have 2 [Icasa filament lightbulbs](https://www.zigbee2mqtt.io/devices/ICZB-B1FC60_B3FC64_B2FC95_B2FC125.html) above the diner table.
These 2 light bulbs are grouped in one group, everything is working fine except that both light bulbs are flashing simultaneous sometimes.
This is a very short flash, something like 200mS and the interval is randomly. Sometimes ones a couple of hours and sometimes 3 times 15 minutes.
If the lights are off the flashing is light on and if the lights are on the the flashing is lights off.
Ever since I have these bulbs my zigbee2mqtt running on a Pi-zero was stopping working a couple of times, the CC2538 pi-hat was not found any more, reading around the CC2538 is not the best nowadays. To overcome the adapter problems it's replaced now with an CC2652P pi-hat.
But the flashing of the bulbs is still there. :-(
Do you have some tip and tricks to solve this problem? 

Z2M - V1.26.0 , CC2652P - CC1352P2_CC2652P_launchpad_coordinator_20220219.hex

Edit:
Change the log to debug and here's is the log from 13:36 when the light where flashing again. Look like the bulbs (eetafel) are asking for an OTA update?
`info  2022-08-24 12:35:14: MQTT publish: topic '/vm/zigbee2mqtt/Lichtsensorachter', payload '{"battery":100,"illuminance":29128,"illuminance_lux":818,"linkquality":43,"voltage":3300}'
debug 2022-08-24 12:36:07: Received Zigbee message from 'Eettafel2', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":537138709,"imageType":4364,"manufacturerCode":4137}' from endpoint 1 with groupID 0
debug 2022-08-24 12:36:07: Device 'Eettafel2' requested OTA
debug 2022-08-24 12:36:13: Received Zigbee message from 'Eettafel1', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":537138709,"imageType":4364,"manufacturerCode":4137}' from endpoint 1 with groupID 0
debug 2022-08-24 12:36:13: Device 'Eettafel1' requested OTA
debug 2022-08-24 12:37:08: Received Zigbee message from 'Eettafel2', type 'commandQueryNextImageRequest', cluster 'genOta', data '{"fieldControl":0,"fileVersion":537138709,"imageType":4364,"manufacturerCode":4137}' from endpoint 1 with groupID 0`

**Comment by Koenkk** [comment]
@EvertDekker the lights are indeed asking for a firmware update but this shouldn't make the bulbs flash so there is definitely something bugged in the firmware of these bulbs. Maybe a firmware update would fix it but we don't have the files, maybe you can ask the manufacturer if they want to provide these.

**Reply by EvertDekker** [reply]
Thanks for your reply.
I found issue #4561 " Occasional flickering of iCasa lamps  " that looks like my problem.
The solutions suggested here I will try.
Will also ask the supplier for a firmware update. Some people in the above issue already tried that, but I will give it a try to.
